spec/support/models/system_role.rb

```diff
@@ -0,0 +1,7 @@
+class SystemRole
+ include Mongoid::Document
+
+  # NOTE: this model is for test purposes only. It is not recommended that you
+  # store Mongoid documents in system collections.
```

Comment:
Adding a note here in case someone stumbles upon this code in the wild and thinks this is a reasonable pattern haha

Reply:
:+1: